README.md

```diff
@@ -22,3 +24,19 @@ npm start
 The result will be in `./generated_trusted_nodes.json` file.
 
 If you hit the API limit, you can also add your infura api key to line 5 in the [src/index.ts].
+
+## Airdrop balances
+
+### Methodology
+The final airdrop reward has been calculated as follows: the awarded addresses are all beneficiaries who cashed out cheque from the trusted nodes between the blocks `4283397` and `5130433` including. The total gBZZ amount an address cashed from the trusted nodes has been summed and devided by the total gBZZ amount cashed from trusted nodes by all the addresses. This ratio then indicates how big portion of the 1 000 000 BZZ given awarded address will receive.
+
+### Data sets
+- Find the final set of addresses and the BZZ balances that they will be awarded [here](./airdrop_rewards.json).
+
+## Extra airdrop
+
+### Methodology
+Due to the amount of participants, the airdrop was extended with an extra amount of 0.1 BZZ to each address that deployed a chequebook through an official chequebook factory and cashed out from any other chequebook deployed through an official chequebook factory.
```

Comment:
```suggestion
Due to the amount of participants, the airdrop was extended with an extra amount of 0.1 BZZ to each address that deployed a chequebook through an official chequebook factory and cashed out from any chequebook deployed through an official chequebook factory.
```

Reply:
I'd prefer to keep it more specific here. Not sure you can technically cashout from your own chequebook, but IMHO we should keep this specific

Reply:
I kind of like the `other` in this sentence, as now it may imply that you could cash out from yourself (which might not be possible, but at least it's clear, right?)

Reply:
it wasn't clear to me what this was saying initially but i understand now, advise to leave it in